content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: delfin
+description: "delfin deployment on kubernetes guide."
+tags: ["installation guide", "delfin"] 
+---
+
+## Delfin deployment on kubernetes cluster
+Here are the steps to deploy delfin on a kubernetes  cluster 
+
+### Pre-config (Ubuntu 18.04)
+The installation is tested on Ubuntu 18.04, please make sure to use the right one.
+Root user is suggested for the installation.
+
+* packages
+
+Install following packages:
+```bash
+apt-get install -y libltdl7 libseccomp2 git curl wget make
+```
+
+* docker
+Install docker:
+```bash
+wget https://download.docker.com/linux/ubuntu/dists/xenial/pool/stable/amd64/docker-ce_18.03.1~ce-0~ubuntu_amd64.deb
+dpkg -i docker-ce_18.03.1~ce-0~ubuntu_amd64.deb 
+```
+
+* Kind
+Install kind:
+```bash
+curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/v0.11.1/kind-linux-amd64
+chmod +x ./kind
+mv ./kind /some-dir-in-your-PATH/kind
+```
+Create a kubernetes cluster:
+```bash
+Kind create cluster
+```
+Check created cluster:
+Example:
+```bash
+root@proxy:~# kind get clusters
+kind
+```
+* kubectl
+ Install kubectl:
+ 
+ Download the latest release with the command:
+```bash
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+```
+ Note:
```

Comment:
You can use the note shortcode to insert important notes. 
You can check the main installation guide [here](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/) and refer to the various kinds of notes, info, and warnings. The shortcode can be found in the code 

https://github.com/sodafoundation/documentation/blob/b14b7d31c2c002b7d5eeacbd5135dfcd55bcb327/content/soda-gettingstarted/installation-using-ansible.md?plain=1#L10

https://github.com/sodafoundation/documentation/blob/b14b7d31c2c002b7d5eeacbd5135dfcd55bcb327/content/soda-gettingstarted/installation-using-ansible.md?plain=1#L78

Reply:
gone through